selfdrive/controls/controlsd.py

```diff
@@ -56,6 +57,9 @@ def __init__(self) -> None:
     elif self.CP.lateralTuning.which() == 'torque':
       self.LaC = LatControlTorque(self.CP, self.CI)
 
+    data_services = list(self.sm.data.keys()) + ['selfdriveStateSP']
+    self.sm = messaging.SubMaster(data_services, poll='selfdriveState')
```

Comment:
for my own knowledge, what does this line do exactly? I've never seen/used the `poll` myself

Reply:
`poll` sets the update frequency inside of `SubMaster`, which for `selfdriveState` is at 100 Hz.